NEWS.md

```diff
@@ -3,6 +3,7 @@
 ## New Features
 
  - Variables labels are applied to datasets by `metacore`/`metatools`.
+ - ADPC, ADPPK and ADPP are updated to handle urine records.
```

Comment:
Shall we start adding issue numbers here? Maybe can do so for the other bullet pts as well.

Reply:
good point, updated